### ISSUE TITLE
Do not use mainnet fork for testing.  Start from scratch, deploy mock tokens, etc

### ISSUE THREAD

**Issue by alexcos20** [oceanprotocol/contracts#478]
**Describe the bug**
Contracts test are depending on hardhat fork of mainnet.
Since that is failing now, all tests are failing.

**Expected behavior**
If you need a token, deploy it using mocks.
DO NOT depend on any fork

**Comment by DMats** [comment]
To whoever creates the PR for this, I think there is also a minor issue in `deploy-docker.sh`. In commit: https://github.com/oceanprotocol/contracts/commit/30f25c7fb05539e20bf1ec75b794a284ab8b0da3, the source argument of `cp` was changed to add `/addresses/` but the `if` condition was not changed.

```
    if [ -e /ocean-contracts/address.json ]   # <- path is wrong, should be ocean-contracts/addresses/address.json
        then cp -u /ocean-contracts/addresses/address.json /ocean-contracts/artifacts/
    fi
```

**Comment by alexcos20** [comment]
Yeah, it's a typo there, but really it does not matter, because there is nothing to reuse in barge. It will always create new addresses

**Comment by DMats** [comment]
@alexcos20 I see what you're saying. Nonetheless, I think this typo should be fixed to eliminate the benign error it causes, even if it doesn't share the same severity as the rest of this issue:

```
ocean-contracts_1  | Error: ENOENT: no such file or directory, open '/ocean-contracts/artifacts/address.json'
ocean-contracts_1  |     at Object.openSync (fs.js:497:3)
ocean-contracts_1  |     at Object.readFileSync (fs.js:393:35)
ocean-contracts_1  |     at main (/ocean-contracts/scripts/deploy-contracts.js:122:36)
ocean-contracts_1  |     at processTicksAndRejections (internal/process/task_queues.js:95:5) {
ocean-contracts_1  |   errno: -2,
ocean-contracts_1  |   syscall: 'open',
ocean-contracts_1  |   code: 'ENOENT',
ocean-contracts_1  |   path: '/ocean-contracts/artifacts/address.json'
ocean-contracts_1  | }
```

**Comment by alexcos20** [comment]
yeah, already fixed in #479

**Comment by alexcos20** [comment]
removed critical because I fixed it in https://github.com/oceanprotocol/contracts/pull/480
